airflow/dag_processing/manager.py

```diff
@@ -586,15 +586,15 @@ def _run_parsing_loop(self):
                     elif agent_signal == DagParsingSignal.END_MANAGER:
                         if span.is_recording():
                             span.add_event(name="end")
-                        self.end()
-                        sys.exit(os.EX_OK)
-                    elif agent_signal == DagParsingSignal.AGENT_RUN_ONCE:
-                        # continue the loop to parse dags
-                        pass
-                    elif isinstance(agent_signal, CallbackRequest):
-                        self._add_callback_to_queue(agent_signal)
-                    else:
-                        raise ValueError(f"Invalid message {type(agent_signal)}")
+                    self.end()
+                    sys.exit(os.EX_OK)
+                elif agent_signal == DagParsingSignal.AGENT_RUN_ONCE:
+                    # continue the loop to parse dags
+                    pass
+                elif isinstance(agent_signal, CallbackRequest):
+                    self._add_callback_to_queue(agent_signal)
+                else:
+                    raise ValueError(f"Invalid message {type(agent_signal)}")
```

Comment:
Is this change intentional? Why is it needed?

Reply:
No reverted it. It was mistake in rebase